src/implementations/twig/components/site-header/__snapshots__/site-header.test.js.snap

```diff
@@ -4723,18 +4755,24 @@ exports[`Site Header EU renders correctly 1`] = `
             class="ecl-link ecl-link--standalone ecl-site-header__logo-link"
             href="/example"
           >
-            <img
-              alt="European Union flag"
-              class="ecl-site-header__logo-image ecl-site-header__logo-image-mobile"
-              src="/condensed-version/positive/logo-eu--en.svg"
-              title="European Union"
-            />
-            <img
-              alt="European Union flag"
-              class="ecl-site-header__logo-image ecl-site-header__logo-image-desktop"
-              src="/standard-version/positive/logo-eu--en.svg"
+            <picture
+              class="ecl-picture ecl-site-header__picture"
               title="European Union"
-            />
+            >
+              <source
+                media="(min-width: 996px)"
+                srcset="/standard-version/positive/logo-eu--en.svg"
+              />
+              <source
```

Comment:
this second source is not needed. By default it will take the image in the img tag (mobile) and switch to desktop